dags/utils/landsat_scenes_identifying_logic.py

```diff
@@ -93,10 +115,12 @@ def filter_africa_location_from_gzip_file(file_path: Path, production_date: str)
     logging.info("Start Filtering Scenes by Africa location, Just day scenes and date")
     logging.info(f"Unzipping and filtering file according to Africa Pathrows")
     for row in read_big_csv_files_from_gzip(file_path):
+
         if (
             # Filter to skip all LANDSAT_4
             row.get("Satellite")
             and row["Satellite"] != "LANDSAT_4"
+            and row["Satellite"] != "4"
```

Comment:
USGS has changed the way that they send **Satellite** for landsat 7 and 8 they send just the number, so before they change landsat 4 as well I added that